lib/davinci_dtr_test_kit/docs/dtr_smart_app_suite_description_v201.md

```diff
@@ -90,26 +103,72 @@ to make requests against Inferno. This does not include the capability to render
+- **Standalone launch sequence**: use requests in the `SMART App Launch` folder during 
+  tests **1.1.1.01** or **2.1.1.01** to simulate the SMART Launch flow and obtain an access
+  token to use for subsequent requests. See the collection's Overview for details on the
+  access token's generation.
+- **1.1** *Static Questionnaire Workflow*: use requests in the `Static Dinner` folder
+  - **1.1.1.01** *Invoke the DTR Questionnaire Package operation*: submit request `Questionnaire Package for Dinner (Static)` while this test is waiting.
+  - **1.1.3.01** *Save the QuestionnaireResponse after completing it*: submit request `Save QuestionnaireResponse for Dinner (Static)` while this test is waiting. If you want to see a failure, submit request `Save QuestionnaireResponse for Dinner (Static) - missing origin extension` instead.
+- **2.1** *Respiratory Assist Device Questionnaire Workflow*: use requests in the `Respiratory Assist Device` folder
+  - **2.1.1.01** *Invoke the DTR Questionnaire Package operation*: submit request `Questionnaire Package for Resp Assist Device` while this test is waiting.
+  - **2.1.3.01** *Save the QuestionnaireResponse after completing it*: submit request `Save Questionnaire Response for Resp Assist Device` while this test is waiting. If you want to see a failure, submit request `Save Questionnaire Response for Resp Assist Device - unexpected override` instead.
+
+## Configuration Details
+
+### `fhirContext` and available instances
+
+Once they have launched, DTR SMART Apps obtain details that drive their retrieval of questionnaires
+and relevant clinical data from the payer and the EHR from [context that is passed with
+the access token](https://hl7.org/fhir/smart-app-launch/STU2.1/scopes-and-launch-context.html)
+provided by the EHR. Inferno cannot know ahead of time what information to provide and
+what instances to make available to direct the app under test to request and render a
+particular questionnaire.
+
+Therefore, use of this test suite requests that the tester provide this information so that the
+app can demonstrate its capabilities based on whatever business logic is present. These tests
+currently support 2 context parameters that contain references to instance in the EHR and provides
+testers with a way to provide those instances to Inferno so it can serve them to the app. These are
+controlled by the following inputs present on each group associated with a questionnaire:
+
+- `SMART App Launch Patient ID`: provide an id for the subject Patient FHIR instance.
+- `SMART App Launch fhirContext`: provide a json object containing FHIR references to instances
+  relevant to the DTR workflow, e.g. 
+  `[{reference: 'Coverage/cov015'}, {reference: 'DeviceRequest/devreqe0470'}]`. This will be included
+  under the `fhirContext` key of the token response.
```

Comment:
This is hard to read

```suggestion
  relevant to the DTR workflow, e.g. 

  ```
  [{reference: 'Coverage/cov015'}, {reference: 'DeviceRequest/devreqe0470'}]
  ```

  This will be included under the `fhirContext` key of the token response.
```

Reply:
committed this change